astropy/io/ascii/ipac.py

```diff
@@ -156,6 +149,14 @@ def process_keyword_value(val):
                     if val:
                         table_meta['comments'].append(val)
 
+    def get_col_type(self, col):
+        for (col_type_key, col_type) in self.col_type_list:
+            if col_type_key.startswith(col.raw_type):
```

Comment:
We should convert to lowercase here. The standard encourages lower case,  so it's fine to use lower case in writing, but upper case is allowed in files. The ipac document says "For compatibility with older applications, it is best to use lower case."